src/span-data.ts

```diff
@@ -18,7 +18,8 @@ import * as crypto from 'crypto';
 import * as util from 'util';
 
 import {Constants, SpanDataType} from './constants';
-import {SpanData as SpanData} from './plugin-types';
+import * as types from './plugin-types';
```

Comment:
Why not just add `RootSpanData` to the import statement below? You don't seem to use anything other than those three (SpanData, SpanOptions, and RootSpanData), do you?

Reply:
It conflicts with `RootSpanData` defined in this file. I couldn't think of any way to avoid this at the time.

Come to think of it, OpenCensus calls the write-only object "Span" (and SpanData refers to the read-only object given to exporters). I'll add a TODO to change this.